plugins/woocommerce/client/admin/package.json

```diff
@@ -183,7 +183,6 @@
 		"await-exec": "^0.1.2",
 		"babel-jest": "27.5.x",
 		"babel-loader": "^8.3.0",
-		"babel-plugin-transform-class-properties": "^6.24.1",
```

Comment:
https://www.npmjs.com/package/babel-plugin-transform-class-properties: es2015 class properties support, unnecessary at this point; last update 8 years ago.